src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -899,6 +899,9 @@ ERT_CLIB_SUBMODULE("torque_driver", m) {
     auto py_path = pathlib.attr("Path")(DEFAULT_QSTAT_CMD.c_str());
     m.add_object("DEFAULT_QSTAT_CMD", py_path);
 
+    py::bind_vector<std::vector<std::string>>(m, "List");
```

Comment:
This line can only be put in one place, I cannot place it in every driver. So where should it be moved to? [help needed]

Reply:
Turned out the line was not needed. Now removed!